src/app-layout/visual-refresh-toolbar/toolbar/drawer-triggers.tsx

```diff
@@ -112,15 +112,18 @@ export function DrawerTriggers({
               ariaLabel={splitPanelToggleProps.ariaLabel}
               ariaControls={splitPanelToggleProps.controlId}
               ariaExpanded={splitPanelToggleProps.active}
-              className={clsx(styles['drawers-trigger'], splitPanelTestUtilStyles['open-button'])}
+              className={clsx(
+                styles['drawers-trigger'],
+                testutilStyles['drawers-trigger'],
```

Comment:
What is the purpose of the additional test class? Can you reuse the existing one (`splitPanelTestUtilStyles['open-button']`)?

Reply:
without this class, the tool, `wrapper.findDrawerTriggerById("slide-panel").toSelector()` will not work as it will not generate the right selector (for this it would be `body [class*="awsui_root_1fj9k"] [class*="awsui_drawers-trigger_1fj9k"][data-testid="awsui-app-layout-trigger-slide-panel"]`)